app/components/primer/auto_complete.rb

```diff
@@ -5,15 +5,21 @@ module Primer
   class AutoComplete < Primer::Component
     status :beta
 
+    DEFAULT_TAG = :"auto-complete"
```

Comment:
Based on [this conversation](https://github.com/primer/view_components/issues/491#issuecomment-827845486), wonder if we should explicitly have:

```
TAG_OPTIONS = [DEFAULT_TAG].freeze
```
and for the slot tags: 
```
INPUT_TAG_OPTIONS = [DEFAULT_INPUT_TAG].freeze
RESULTS_TAG_OPTIONS = [DEFAULT_RESULTS_TAG].freeze
```

It felt a bit overkill since there's only one allowed, fixed tag but maybe having an explicit allowed list of tags is good for consistency ¯\_(ツ)_/¯  open to thoughts